hack/tools/containerd-puller/go.mod

```diff
@@ -0,0 +1,50 @@
+module k8s.io/registry.k8s.io/hack/tools/containerd-puller
+
+go 1.19
+
+require github.com/containerd/containerd v1.7.0
+
+require (
```

Comment:
didn't want to pull all of this into the main module, so added another tools module ...

see below for: probably long term we can see about gettin `--no-unpack` in `ctr images pull` instead

Reply:
removed in favor of `ctr content fetch` TIL!